Core/Core/API/AssignmentListRequestable.swift

```diff
@@ -75,7 +74,7 @@ public struct AssignmentListRequestable: APIGraphQLRequestable {
 
     static let operationName = "AssignmentList"
     static let query = """
-        query operationName($courseID: ID!, $pageSize: Int!, $cursor: String, $filter: AssignmentFilter) {
+        query \(operationName)($courseID: ID!, $pageSize: Int!, $cursor: String, $filter: AssignmentFilter!) {
```

Comment:
Why did this change to `AssignmentFilter!` ?

Reply:
because the API throws an error when it gets a null, so might as well put that in the type.

Reply:
Is there a reason for it not to be "!"?

Reply:
Ah, i'm sorry, thought this was swift code.  It's graphql, my apologies.